feature/CourseView/Sources/CourseView/ExerciseTab/ExerciseDetailView.swift

```diff
@@ -165,6 +226,19 @@ public struct ExerciseDetailView: View {
     }
 }
 
+private struct ExerciseDetailCell<Content: View>: View {
+    let descriptionText: String
+    @ViewBuilder let content: Content
+
+    var body: some View {
+        HStack {
+            Text(descriptionText)
+            Spacer()
+            content
+        }.padding(.m)
```

Comment:
```suggestion
        }
        .padding(.m)
```

Modifiers generally start at the new line in project. Then, they all align if there are more than one